src/components/CityDetails/CityDetails.js

```diff
@@ -1,13 +1,43 @@
-import React from "react";
+import React, { Component } from 'react';
+import { getSingleCity } from '../../apiCalls';
 import './CityDetails.css';
+class CityDetails extends Component {
+  constructor() {
+    super()
+    this.state = {
+      city: {},
+      error: '',
+    }
+  }
 
-const CityDetails = () => {
+  componentDidMount = async() => {
+    try {
+      const fetchedCity = await getSingleCity(this.props.id)
+      this.setState({ city: fetchedCity[0] })
+      console.log(this.state.city)
+    } catch {
+      this.setState({ error: "Oops!" })
```

Comment:
This is great.  Love using the async functions with try & catch blocks, and I am looking forward to practicing using them more on my final project for Mod3.  I'm so excited to build something with all my new knowledge!